demos/demo06.py

```diff
@@ -25,8 +25,7 @@
 
 def main():
     colorama.init()
-    # gratuitous use of lambda.
-    pos = lambda y, x: '\x1b[%d;%dH' % (y, x)
+    pos = lambda y, x: Cursor.POS(x,y)
```

Comment:
Please add a space after the comma ([flake8 compliance](https://www.flake8rules.com/rules/E231.html))